csaf-import/src/test/java/io/github/csaf/sbom/retrieval/RetrievedProviderJavaTest.java

```diff
@@ -26,17 +28,21 @@
  * Tests the functionality of <code>RetrievedProvider</code> in Java.
  */
 public class RetrievedProviderJavaTest {
-    private static final CsafLoader loader = new CsafLoader(TestUtilsKt.mockEngine());
+    @BeforeAll
+    public static void setup() {
+        //noinspection KotlinInternalInJava
```

Comment:
left over comment or is that actually doing something?

Reply:
The Java package is considered non-internal from a Kotlin perspective. Without this comment, this is a "soft" error within IDEA.